perllib/FixMyStreet/App/Controller/Reports.pm

```diff
@@ -588,6 +588,12 @@ sub load_and_group_problems : Private {
         }
     }
 
+    my $bbox = $c->get_param('bbox');
```

Comment:
$c is available inside the cobrand as $self->{c}, so easier to look up these two vars there and not bother the controller about it?

Reply:
Ah didn't know that! Yes much easier to just do it in the cobrand, ta.